scriptmodules/emulators/hypseus/hypinput.ini

```diff
@@ -0,0 +1,58 @@
+# Sample hypinput.ini
+# All key options listed
+# Hypseus uses SDL2 Keycodes
+#
+# The first two entries are SDL2 keyboard codes or names (0 for "none")
+#
+# Find SDL2 keyboard code information here:
+# https://gitlab.com/DirtBagXon/hypseus-singe/-/blob/main/doc/keylist.txt

```

Comment:
Should maybe change the link to <https://github.com/DirtBagXon/hypseus-singe/blob/master/doc/keylist.txt> or directly to <https://wiki.libsdl.org/SDLKeycodeLookup>.

Reply:
Will do, although that https://wiki.libsdl.org link keeps disappearing which is why I host locally :)